addons/sale_stock/tests/test_sale_stock.py

```diff
@@ -21,7 +21,7 @@ def setUpClass(cls):
             'is_storable': True,
         })
 
-    def _get_new_sale_order(self, amount=10.0, product=False):
+    def _get_new_sale_order(self, amount=10.0, product=False, sol_vals={}):
```

Comment:
We should avoid to use mutable as default parameter's value ([reading about this issue](https://stackoverflow.com/questions/26320899/why-is-the-empty-dictionary-a-dangerous-default-value-in-python))
```suggestion
    def _get_new_sale_order(self, amount=10.0, product=False, sol_vals=False):
```
And then, you can do that instead:
```python
sol_vals = sol_vals or {}
```